geonode/qgis_server/helpers.py

```diff
@@ -409,8 +409,8 @@ def thumbnail_url(bbox, layers, qgis_project, style=None, internal=True):
         'REQUEST': 'GetMap',
         'BBOX': bbox,
         'SRS': 'EPSG:4326',
-        'WIDTH': '250',
-        'HEIGHT': '250',
+        'WIDTH': '240',
+        'HEIGHT': '180',
```

Comment:
@boney-bun can you first explain why it has to be this value?
IMO, we should find a way to request a square thumbnail, then the aspect ratio can be tweak in the UI using CSS.

Reply:
@lucernae the numbers is coming from the geoserver. i tried to imitate what the geoserver do with the thumbnail as in https://github.com/kartoza/geonode/blob/master-qgis_server/geonode/geoserver/helpers.py#L1651